common/persistence/historyV2Store.go

```diff
@@ -73,12 +73,19 @@ func (m *historyV2ManagerImpl) ForkHistoryBranch(request *ForkHistoryBranchReque
 	if err != nil {
 		return nil, err
 	}
-
+	shardID, err := getShardID(request.ShardID)
+	if err != nil {
+		m.logger.WithError(err).Error("shardID is not set in fork history branch operation")
```

Comment:
We don't need to log this error. The application layer should already have log. We only log something in persistence layer when we need to log some additional information for debugging.